_cities/milan.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Milan
+location: "Milan, Italy"
+tagline: "Pizza, code and fine wine!"
+mailing_list: team.milan
+does:
+  workshops
+members:
+  - alberto: Alberto Massa
+  - giorgia
+  - giorgio: Giorgio Mazzucchelli
+  - pietro909
```

Comment:
Does this account actually exist?  Members without an opentechschool.org account should be added like the above, with `<image name>: <full name>` (otherwise it's going to add `pietro909@opentechschool.org` as a contact address for you.)